Source/DafnyTestGeneration/Inlining/InliningTranslator.cs

```diff
@@ -43,6 +44,8 @@ public static class InliningTranslator {
     boogieProgram = MergeBoogiePrograms(boogiePrograms);
     // Finally, create bodies for the Call$$ procedures that call out to Impl$$ procedures
     boogieProgram = new AddImplementationsForCallsRewriter(options).VisitProgram(boogieProgram);
+    // Remove proof dependency ids because they lead to errors when attempting inlining
```

Comment:
I was thinking of making the `AddProofDependencyId` method in `ProofDependencyManager` virtual and then extending the class and overriding the method so that no attribute is added but I don't know if that wouldn't break anything? Then all one would have to do to disable the annotation is to set the program's proof dependency manager to a new instance of that new class.

Reply:
Ultimately, the `{:id}` attributes are only used when the Boogie `-trackVerificationCoverage` flag is enabled. I think the easy thing to do would be to have the code to add `{:id}` attributes check that option, and not add them if it isn't set.